### PR TITLE
Revert to default when changing subscribable.

### PR DESCRIPTION
**Pull request by aboodman**
See https://discord.com/channels/830183651022471199/1082164343715663923/1082164343715663923

### CODE REVIEW

File: src/index.test.tsx [hunk 1]
```diff
@@ -141,3 +141,56 @@ test('returning undefined', async () => {
...
+  render(<A rep={rep2} val="b" res={r2} />, div);
+  await p2;
+  await sleep(1);
+  expect(div.textContent).to.equal('b');
+
+  const {resolve: r3} = resolver();
+  render(<A rep={undefined} val="c" res={r3} />, div);
+  await sleep(1);
+  expect(div.textContent).to.equal('');
+});
```
Comment (arv) [inline src/index.test.tsx:196]: I realize we are not closing the Replicache instances in this test. Can you add that while you are here?

**Review by arv — APPROVED** [review]
LGTM

This new behavior feels better to me.

**Comment by user72356** [comment]
Can you guys merge this with the main branch soon please :-)

**Comment by user72356** [comment]
> @user72356 Done as 2.11.0

Thank you!

**Pull request merged**